phpBB/includes/functions_admin.php

```diff
@@ -534,6 +534,19 @@ function move_topics($topic_ids, $forum_id, $auto_sync = true)
 	}
 
 	$table_ary = array(TOPICS_TABLE, POSTS_TABLE, LOG_TABLE, DRAFTS_TABLE, TOPICS_TRACK_TABLE);
+
+	/**
+	 * Perform additional actions before topics move
+	 *
+	 * @event core.move_topics_before_query
+	 * @var	array	table_ary	Array of tables from which forum_id will be updated for all rows that hold the moved topics
+	 * @since 3.1.5-RC1
+	 */
+	$vars = array(
+			'table_ary',
```

Comment:
same here.
Please add all method parameters and other relevant variables to the event.